java/samples/sample-code/src/main/java/com/microsoft/semantickernel/syntaxexamples/Example13_ConversationSummarySkill.java

```diff
@@ -186,9 +186,10 @@ private static Kernel initializeKernel(Config.ClientType clientType) throws IOEx
 
         KernelConfig kernelConfig =
                 new KernelConfig.Builder()
-                        .addTextCompletionService("text-davinci-003", kernel -> textCompletion)
                         .build();
 
-        return SKBuilders.kernel().setKernelConfig(kernelConfig).build();
+        return SKBuilders.kernel()
+                .withKernelConfig(kernelConfig)
+                .withAIService("text-davinci-003", textCompletion, true, TextCompletion.class).build();
```

Comment:
Please avoid using `text-davinci-003`. OpenAI and most users are favouring using chat completion models.

Reply:
FYI @johnoliver